queries/haskell/highlights.scm

```diff
@@ -116,6 +116,8 @@
 ((signature (forall (context (fun)))) . (function (variable) @function))
 
 (exp_infix (variable) @operator)  ; consider infix functions as operators
+(exp_section_right (variable) @operator) ; partially applied infix functions (sections) also get highlighted as operators
```

Comment:
At the moment, operators are either symbols `+`/`-` (`@operator`) or keywords (`new`, `delete` aka. `@keyword.operator`. This seems to open a new category. Could still be fine as `@operator`